modules/3d-tiles/src/lib/ion/ion.js

```diff
@@ -35,10 +34,7 @@ export async function getIonAssets(accessToken) {
   assert(accessToken);
   const url = CESIUM_ION_URL;
   const headers = {Authorization: `Bearer ${accessToken}`};
-  const response = await fetch(url, {headers});
-  if (!response.ok) {
```

Comment:
More conceptual thought: While we hard code `fetchFile` here for now, my sense is that we generally want to support custom `fetch` function, and after this change loaders may still want to check `response.ok` if custom fetch is passed in, since we shouldn't rely on extra functionality in fetch.